releases/v0.5.0.md

```diff
@@ -0,0 +1,70 @@
+# Release Notes for v0.5.0
+Release Date: 2023-03-14
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+
+The following platforms are untested or partially supported:
+- Intel SGX
+- AMD SEV-ES
+- IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+
+The following platforms are in development:
+- AMD SEV-SNP
+
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * AMD SEV-ES is not tested in the CI.
+  * Image signature validation has not been tested with AMD SEV.
+  * s390x does not support cosign signature validation
+- SELinux is not supported on the host and must be set to permissive if in use.
+- Attestation and key brokering support is still under development
+  * The disk-based key broker client (KBC) is used for non-tee testing, but is not suitable for
+  production, except with encrypted VM images.
+  * Currently, there are two key broker services (KBS) that can be used:
+    - simple-kbs:  simple key broker service for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which
+    Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
```

Comment:
In v0.5.0, generic KBS and supporting KBC can be released. We will integrate them into the guest image/initrd and give the using guide in Quick Start.

cc @sameo @fitzthum @Xynnn007 @jepio

Related issue: https://github.com/kata-containers/kata-containers/issues/6365